src/arrayMethodSort.js

```diff
@@ -6,6 +6,40 @@
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
     // write code here
+    let compare = compareFunction;
+
+    if (!compareFunction) {
+      compare = function(a, b) {
+        if (('' + a) > ('' + b)) {
+          return 1;
+        } else if (('' + a) === ('' + b)) {
+          return 0;
+        } else {
+          return -1;
+        }
```

Comment:
Try to avoid **else** construction when you can, it will make code easier to read
```suggestion
        if (('' + a) > ('' + b)) {
          return 1;
        } 
        
        if (('' + a) === ('' + b)) {
          return 0;
        } 
        
        return -1;
 
```